example/pipeline/pipeline_pdf.ipynb

```diff
@@ -68,7 +68,7 @@
    "source": [
     "import os\n",
     "import pandas as pd\n",
-    "from uniflow.pipeline import MultiProcessPipeline\n",
+    "from uniflow.pipeline import MultiThreadPool\n",
```

Comment:
THIS IS A VERY BAD IDEA! This is not a MultiThreadPool, but a streaming processing pipeline. You should name it `Pipeline` or `MultiStagePipeline`.

Reply:
or `MultiFlowsPipeline` as you suggested @goldmermaid

Reply:
You mentioned Multi thread pool here: https://github.com/CambioML/uniflow/pull/58/files/01fc9603a192dd943936e5970fbc624188bd830d#r1433343534